test/examples_test.go

```diff
@@ -14,7 +14,7 @@ func TestCoinPricesFromCoingecko(t *testing.T) {
 	requiredEnvVars := []string{
 		"YOUR_COINGECKO_API_KEY",
 	}
-	NewTestscriptTest(t, projectDir).
+	NewTestscriptTest(t, projectDir, "").
```

Comment:
Now `TestscriptTest` expects a "save directory" where the `saveFile` command will save output files. Even if passed as a relative path it will get converted to an absolute path so the testscript is able to save the file to a location outside of its temporary work directory.

If no save directory is supplied, then the `saveFile` command simply prints a warning that the file was not saved and the test moves on without error. This behavior is so that we can reuse testscript scripts without requiring that they always write output files (we want to save the output for combination tests but not for normal tests).